### ISSUE TITLE
I am running Android and this simple test code:

### ISSUE THREAD

**Issue by thejobsmarket** [cunarist/rinf#351]
Still cannot access the file system from Rust
I probably need to do something to allow disk access for the rust process but I don't know what, so moving this back to being  issues
I am running Android and this simple test code:

        let paths = fs::read_dir("./").unwrap();
        for path in paths {
            debug_print!("Name: {}", path.unwrap().path().display());
        }

  I/flutter (32666): A panic occurred in Rust.
I/flutter (32666): panicked at native/hub/src/plugin_loader.rs:12:40:
I/flutter (32666): called `Result::unwrap()` on an `Err` value: Os { code: 13, kind: PermissionDenied, message: "Permission denied" }

_Originally posted by @thejobsmarket in https://github.com/cunarist/rinf/discussions/348#discussioncomment-9662532_

**Comment by temeddix** [comment]
Hi, could you try this solution?

- https://stackoverflow.com/questions/50561737/getting-permission-to-the-external-storage-file-provider-plugin

Please tell me if you're still facing the same problem.

**Comment by thejobsmarket** [comment]
I gave storage, external storage and all type of other access permissions; android:requestLegacyExternalStorage="true" but rinf keeps on crashing. Worst thing is that it crashes and afterwards does not respond any more:
directory: /
I/flutter (30201): trying: .
I/flutter (30201): A panic occurred in Rust.
I/flutter (30201): panicked at native/hub/src/plugin_loader.rs:26:36:
I/flutter (30201): called `Result::unwrap()` on an `Err` value: Os { code: 13, kind: PermissionDenied, message: "Permission denied" }
I/flutter (30201):    0: <unknown>
I/flutter (30201):    1: <unknown>
I/flutter (30201):    2: <unknown>
I/flutter (30201):    3: <unknown>
I/flutter (30201):    4: <unknown>
When asked the directory, it is in /, I tried ., all the assets directories in pubspec.yaml,... 
Would it be helpful to prepare a special flutter project so the bug can be reproduced?
